resources/views/equipment/index.blade.php

```diff
@@ -22,7 +22,7 @@
     <div class="well">
         <h3>View tools, manuals, and book inductions</h3>
         For changes to the information on the equipment pages please contact someone on
-        the <a href="https://members.hacman.org.uk/groups/equipment">equipment</a> team
+        the <a href="<?=URL::to('/')?>/groups/equipment">equipment</a> team
```

Comment:
We can use route helpers to generate links, rather than having to hard code url structures.

Also we shouldn't need to use short echo tags or raw PHP in these templates, check out docs on [Blade templating](https://laravel.com/docs/5.3/blade).

```suggestion
        the <a href="{{ routes('groups.show', 'equipment' }}">equipment</a> team
```

(untested)

Reply:
tested and works fine